tests/resources/Harbor-Pages/ToolKit.robot

```diff
@@ -37,27 +37,37 @@ Delete Fail
 Filter Object
 #Filter project repo user tag.
     [Arguments]    ${kw}
-    Click Element  xpath=//hbr-filter//clr-icon
-    Input Text   xpath=//hbr-filter//input  ${kw}
-    Sleep  1
+    ${element}=  Set Variable  xpath=//hbr-filter//clr-icon
+    Wait Until Element Is Visible And Enabled  ${element}
+    Click Element    ${element}
+    ${element}=  Set Variable  xpath=//hbr-filter//input
+    Wait Until Element Is Visible And Enabled  ${element}   
+    Input Text   ${element}  ${kw}
+    Sleep  3
```

Comment:
Is the `Sleep` necessary?

Reply:
I will file a topic on this question, because it's a search operation, for now there is no sign for completion of search operation, I plan to ask GUI developer to hide an invisible element as a sign, before that I have to sleep 3 seconds.